src/tests/classic/stage_2.rs

```diff
@@ -0,0 +1,135 @@
+use clvmr::allocator::Allocator;
+use std::rc::Rc;
+
+use crate::classic::clvm_tools::binutils::{assemble_from_ir, disassemble};
+use crate::classic::clvm_tools::ir::reader::read_ir;
+use crate::classic::clvm_tools::stages::stage_2::compile::{
+    do_com_prog, try_expand_macro_for_atom,
+};
+use crate::classic::clvm_tools::stages::stage_2::helpers::brun;
+use crate::classic::clvm_tools::stages::stage_2::operators::run_program_for_search_paths;
+
+fn test_expand_macro(
+    allocator: &mut Allocator,
+    macro_data: String,
+    prog_rest: String,
+    macros: String,
+    symbols: String,
+) -> String {
+    let macro_ir = read_ir(&macro_data).unwrap();
+    let macro_source = assemble_from_ir(allocator, Rc::new(macro_ir)).unwrap();
+    let prog_ir = read_ir(&prog_rest).unwrap();
+    let prog_source = assemble_from_ir(allocator, Rc::new(prog_ir)).unwrap();
+    let macros_ir = read_ir(&macros).unwrap();
+    let macros_source = assemble_from_ir(allocator, Rc::new(macros_ir)).unwrap();
+    let symbols_ir = read_ir(&symbols).unwrap();
+    let symbols_source = assemble_from_ir(allocator, Rc::new(symbols_ir)).unwrap();
```

Comment:
Because it's a short circuit used by a test, i don't think the error handling is important.
It's not a test on its own it's a shortcut for writing smaller tests.
If there's a convention for such functions then i can change to that.
Apologies for writing code that isn't clear in that way.

Reply:
it could be annotated with `#[cfg(test)]` it sounds like